sources/web/datalab/static/notebook-app.js

```diff
@@ -95,6 +95,9 @@ define(['static/appbar', 'static/minitoolbar'], function(appbar, minitoolbar) {
       })
     });
 
+    idleTimeout.setupKernelBusyHeartbeat();
+    $('#mainArea').scroll(idleTimeout.notebookScrolled);
```

Comment:
Yea I think you're right about the list page, the sessions page also uses the same code so it'll do the same refresh requests.
We have to think about other pages though, for now we only have the editor and terminal pages, which both sound like the user can spend some time on. Can we make sure we have those covered?